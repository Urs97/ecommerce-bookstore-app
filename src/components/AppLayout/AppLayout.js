import React from 'react';
import './AppLayout.scss';
import Navigation from '../Navigation/Navigation';
import Footer from '../Footer/Footer';

function AppLayout(props) {
    return (
        <React.Fragment>
            <Navigation />
            {props.children}
            <Footer />
        </React.Fragment>
    )
};

export default AppLayout;