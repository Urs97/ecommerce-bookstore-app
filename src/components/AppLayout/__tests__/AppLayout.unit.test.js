import React from 'react';
import {shallow} from 'enzyme';
import AppLayout from '../AppLayout';

describe('AppLayout', () => {
    test('renders empty AppLayout', () => {
        const wrapper = shallow(
            <AppLayout />
        );
        expect(wrapper).toMatchSnapshot();
    });
});