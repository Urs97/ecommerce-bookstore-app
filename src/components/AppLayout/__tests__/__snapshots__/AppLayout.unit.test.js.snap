// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLayout renders empty AppLayout 1`] = `
<Fragment>
  <Navigation />
  <Footer />
</Fragment>
`;
